src/js/pages/Home.js

```diff
@@ -3,105 +3,103 @@ var $ = require('jquery');
 var _ = require('lodash');
 
 import BroadcastMessage from "../components/BroadcastMessage";
-import Events from "../components/Events";
+import EventStore from "../stores/EventStore";
+import NotificationStore from "../stores/NotificationStore";
+import Card from "../components/Card";
 
 import { NotificationStack } from 'react-notification';
-import { OrderedSet } from 'immutable';
 var moment = require('moment');
 
 export default class Featured extends React.Component {
   constructor(props) {
-    super();
-    this.addNotification = this.addNotification.bind(this);
-    this.removeNotification = this.removeNotification.bind(this);
+    super(props);
+    this.getEvents = this.getEvents.bind(this);
+    this.getNotifications = this.getNotifications.bind(this);
     this.state = {
-      messages: [],
-      messageStack: OrderedSet(),
+      notifications: [],
       events: [],
       apiBaseURL: props.route.apiBaseURL,
     };
-  }
+  };
 
-  addNotification (message) {
-    const currTime = moment().unix();
-    if (currTime > message.expirytime) {
-      return;
-    }
-    var self = this;
-    return this.setState({
-      messageStack: this.state.messageStack.add({
-        message: message.message,
-        key: message.id,
-        dismissAfter: (message.expirytime - currTime) * 1000
-      })
-    });
-  }
+  componentDidMount() {
+    NotificationStore.getAll();
+    EventStore.getAll();
```

Comment:
I wonder if any differences will occur with calling store loads in compnentDidMount vs componentWillMount